packages/instantsearch-react/src/impl/HitsPerPage.js

```diff
@@ -4,48 +4,43 @@ import themeable from '../themeable';
 
 import LinkList from './LinkList';
 
-class HitsPerPage extends Component {
+const ThemedLinkList = themeable({
+  root: 'HitsPerPage',
+  item: 'HitsPerPage__item',
+  itemSelected: 'HitsPerPage__item--selected',
+})(LinkList);
+
+export default class HitsPerPage extends Component {
   static propTypes = {
-    applyTheme: PropTypes.func.isRequired,
     hitsPerPage: PropTypes.number.isRequired,
-    createURL: PropTypes.func.isRequired,
-    refine: PropTypes.func.isRequired,
     items: PropTypes.oneOfType([
-      PropTypes.arrayOf(PropTypes.shape({
-        label: PropTypes.string.isRequired,
-        value: PropTypes.number.isRequired,
-      })),
+      PropTypes.arrayOf(PropTypes.object),
```

Comment:
This component doesn't care about the shape of an item, since they get passed directly to the `listComponent`.